README.md

```diff
@@ -63,35 +63,43 @@ Fluture-Express mutates the response object for you, based on a
 specification of what the response should be. This specification is
 captured by the Response sum-type. It has these constructors:
 
-#### <a name="Stream" href="https://github.com/fluture-js/fluture-express/blob/v3.1.0/index.js#L72">`Stream :: (Number, String, NodeReadableStream) -⁠> Response a`</a>
+#### <a name="Stream" href="https://github.com/fluture-js/fluture-express/blob/v3.1.0/index.js#L133">`Stream :: Number -> String -> NodeReadableStream -⁠> Response a`</a>
```

Comment:
I made the line numbers point to the comment for the function in index.js. Also, wasn't sure if blob/v3.1.0 was correct.

Reply:
__README.md__ is automatically updated at release time, so you should revert your changes to this file. Read about [Transcribe][1] if you're curious. :)


[1]: https://github.com/plaid/transcribe

Reply:
@voxbono I think there's been a misunderstanding. I asked for `example/README.md` to be updated, but you probably thought I meant `/README.md`. What @davidchambers said is right: `/README.md` is generated from code comments, and shouldn't be manually modified. It will be automatically built when I release the new version, with line numbers and all.